scripts/source/source_cloud_native_operator.py

```diff
@@ -37,42 +37,62 @@ def index_frontmatter():
                 readingNav = True
             elif readingNav:
                 nav.append(line.replace(".md", ""))
+                if "quickstart.md" in line:
+                    nav.append("  - interactive\n")
 
     return INDEX_FRONTMATTER.format("".join(nav))
 
 
 def process_md(file_path):
-    new_file_path = file_path.replace(".md", ".mdx")
+    new_file_path = file_path.with_suffix(".mdx")
 
     with open(new_file_path, "w") as new_file:
         with open(file_path, "r") as md_file:
             copying = False
-            gh_relative_path = file_path.replace("temp_kubernetes/build/", "src/")
+            quickstart = file_path.name == "quickstart.md"
+            paragraph = 0
+            gh_relative_path = "src/" + str(
+                file_path.relative_to("temp_kubernetes/build/")
+            )
 
             for line in md_file:
-                if copying:
-                    new_file.write(rewrite_yaml_links(line))
-                if line.startswith("#") and not copying:
+                if not line.strip():
+                    paragraph = paragraph + 1
+
+                    if quickstart and paragraph == 2:
```

Comment:
[q] This feels a little fragile. Is it not possible to inject a comment in the original source, then search and replace based on that?

Reply:
It is fragile; I was aiming to keep this fairly simple initially though.

For the future, I'd like to not have any content *in* the import script itself. Comment + keyword + content in the source material would be responsible for tracking docs-specific stuff, e.g.

```yaml

nav:
 - a
#docs-content - q
 - b
```

```markdown
blah blah blah

<!-- docs-content
!!! Admonition
	that should only appear after import
-->

This would offer a lot more flexibility going forward; may be overkill right now.

Reply:
Can you add an issue for improvement, in case we come across another similar use case?

Reply:
Good plan! https://github.com/EnterpriseDB/docs/issues/1224